### DISCUSSION TITLE
Is it possible pass own url to image?

### DISCUSSION THREAD

**Discussion by krajcikondra** [missive/emoji-mart#621] [Q&A] [answered]
Is it possible pass getImageURL props? 

When I use Picker like this
```
import { Picker } from 'emoji-mart';

 <Picker
        onSelect={onSelectEmoji}
        set={emojiSet ?? undefined}
        getImageURL={() => {
          return 'https://myweb.com/emoji-data/sheet_google_64.png';
        }}
      />
```

in Chrome -> Network I look still source https://unpkg.com/emoji-datasource-google@5.0.1/img/google/sheets-256/64.png

Is it correct way to use?

**Comment by EtienneLem** [comment, marked as answer]
You would need to use `getSpritesheetURL` instead of `getImageURL`.

Apologies for the lack of clear documentation, coming soon! 🙏🏼
